packages/ckeditor5-build-helpjuice/src/hjeditor.js

```diff
@@ -277,7 +277,10 @@ HelpjuiceEditor.defaultConfig = {
 			'linkImage',
 			'toggleImageCaption',
 			'imageTextAlternative'
-		]
+		],
+		upload: {
+			types: [ 'jpeg', 'png', 'gif', 'bmp', 'webp', 'tiff', 'svg+xml' ]
```

Comment:
@NicholasJarr you might want to add `jpg` as well and it's good to go.

Reply:
@chille1987

The `jpeg` entry on the array also allows `.jpg` files. I tried it locally and it worked as expected.

Reply:
Ok then, feel free to merge it.